shared/src/business/utilities/pdfGenerator/documentTemplates/PrintableWorkingCopySessionList.jsx

```diff
@@ -0,0 +1,206 @@
+const React = require('react');
+import { PrimaryHeader } from '../components/PrimaryHeader';
+import { ReportsHeader } from '../components/ReportsHeader';
+import { SessionNotesSection } from '../components/SessionNotesSection';
+
+const isMemberCase = formattedCase => {
+  return formattedCase.inConsolidatedGroup && !formattedCase.leadCase;
+};
+
+const orderedFilterMap = [
+  {
+    code: 'setForTrial',
+    name: 'Set For Trial',
+  },
+  {
+    code: 'dismissed',
+    name: 'Dismissed',
+  },
+  {
+    code: 'continued',
+    name: 'Continued',
+  },
+  {
+    code: 'rule122',
+    name: 'Rule 122',
+  },
+  {
+    code: 'aBasisReached',
+    name: 'A Basis Reached',
+  },
+  {
+    code: 'settled',
+    name: 'Settled',
+  },
+  {
+    code: 'recall',
+    name: 'Recall',
+  },
+  {
+    code: 'takenUnderAdvisement',
+    name: 'Taken Under Advisement',
+  },
+  {
+    code: 'statusUnassigned',
+    name: 'Status Unassigned',
+  },
+];
+
+const generateSelectedFilterList = filters => {
```

Comment:
🔧 There is quite a bit of logic in here, my understanding is that in general we want to try and keep as much logic out of view files as possible so they can have a single responsibility and because they aren't tested. Would it be possible to move some of these functions and constants into other files where they can be tested?